### PR TITLE
[AWS QE] add rehearse jobs for each cluster profile

### PR DESCRIPTION
**Pull request by yunjiang29**
add rehearse jobs for each cluster profile, `aws-qe` and `aws-c2s-qe`

### CODE REVIEW

**Comment by yunjiang29** [comment]
@jianlinliu PTAL.

**Comment by jianlinliu** [comment]
Hmm, I meant we should add rehearse jobs for each AWS cluster profile, e.g: one for `aws-qe`, one for `aws-usgov-qe`, one for `aws-c2s-qe`.

File: ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse.yaml [hunk 1]
```diff
@@ -33,13 +33,20 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: installer-rehearse-aws
+- as: installer-rehearse-aws-ipi
```
Comment (jianlinliu) [inline ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse.yaml:36]: suggest to naming it with cluster profile name, e.g: `installer-rehearse-aws-qe`

File: ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse.yaml [hunk 2]
```diff
@@ -33,13 +33,20 @@ resources:
...
   interval: 960h
   steps:
     cluster_profile: aws-qe
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
-    workflow: cucushift-installer-rehearse-aws-ipi-byo-route53
+    workflow: cucushift-installer-rehearse-aws-ipi
+- as: installer-rehearse-aws-ipi-disconnected-private-usgov
```
Comment (jianlinliu) [inline ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse.yaml:43]: suggest to naming it with cluster profile name, e.g: `installer-rehearse-aws-usgov-qe`

**Pull request merged**